.github/workflows/test.yml

```diff
@@ -51,6 +50,67 @@ jobs:
         # need to figure out what happened on mac with the utilities
         run: python -m pymmcore_plus.install -r 20220420
 
+      - name: Install Linux Micro-Manager build tools and dependencies
+        if: runner.os == 'Linux'
+        run: |
+          sudo apt-get update
+          sudo apt-get -y install build-essential autoconf automake libtool autoconf-archive pkg-config
+          sudo apt-get -y install libboost-all-dev
+          sudo apt-get -y install libpcre3-dev  # For SWIG-3
+          sudo apt install -y qt5-default libgl1-mesa-glx xvfb # Qt(5) and headless display dependencies
+          # Below are specific to gh actions https://pytest-qt.readthedocs.io/en/latest/troubleshooting.html#github-actions
+          sudo apt install libxkbcommon-x11-0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-randr0 libxcb-render-util0 libxcb-xinerama0 libxcb-xfixes0
+          /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -screen 0 1920x1200x24 -ac +extension GLX
+
+      - name: Cache SWIG-3
+        if: runner.os == 'Linux'
+        id: cache-swig-3
+        uses: actions/cache@v2
+        with:
+          path: swig-3.0.12
+          key: mmstudio-javadoc-swig-3.0.12
+
+      - name: Build SWIG-3
```

Comment:
i know this is just copied from micro-manager, and this can be played with in a follow up PR, but the linux runner actually [comes with swig 4.0.2-1ubuntu1](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2204-Readme.md).  Do you happen to know if swig 3? is required?

Reply:
annoying :joy:

Reply:
Also it's likely faster to just use conda to install swig-3  that's what I settled on for the build instructions for micromanager